### DISCUSSION TITLE
Separate static file documenation.

### DISCUSSION THREAD

**Discussion by 312-ninja** [erebe/greenclip#109] [Q&A]
The documentation for running a separate static file is very poor. Coming from version 3, I haven't been able to figure out how to get this working properly. Can anyone give me some directions where I can find more info regarding this setup with toml?

**Comment by erebe** [comment]
Hello,
Since version 4 there is no separate file for static history.
It is directly embeded in the config file under the section `static_history`
Just put one element per item of the array.
It follows the toml standard https://toml.io/en/
